### DISCUSSION TITLE
Future Suggestions

### DISCUSSION THREAD

**Discussion by ghost** [TheNewHEROBRINEX/Murder#59] [Ideas]
To make the game much better and cooler, consider adding an shop and top kills and wins leaderboard to make the server much better. For the shop consider adding kinds of swords to buy or more murder chance kind of thing.
I know this will be a little hard so that's why I put a future suggestion.

**Comment by TheNewHEROBRINEX** [comment]
What in-game advantages would have a stone/iron/gold/diamond sword over a wooden one?

**Comment by ghost** [comment]
Extremely Good question. I’m predicting maybe these following 
- Stone: Gives a speed effect when gain a kill.
- Gold: Doubles up the kills but the sword will lose much faster since you need to collect emeralds.
-  Iron: Maybe when the sword is throwing you will teleport there. Maybe a trident effect when you use riptide.
- Diamond: Gains an invisibility effect for 3 seconds when you kill someone with it.

Again these are my future suggestion since it is just to make the game more interesting
